lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2519,6 +2519,9 @@ def test_get_students_features(self):
         Test that some minimum of information is formatted
         correctly in the response to get_students_features.
         """
+        for student in self.students:
```

Comment:
When I was making the code modification below, I first deleted city and country and then ran the test modified in the original PR (#11620). I was disappointed to see that no tests failed because the method being tested wasn't the "get_students_features" method-- it was instead a utility method to convert the value to something that could be returned to get_students_features. Therefore I added this test as well.

Note though that this test is going through the JSON (non-CSV) code path, and therefore I can't get that City and Country are at the end. I don't see any existing tests that are generating the CSV file and looking at its contents, probably because of the complication with the celery instructor task. I didn't think it was worth tackling that sort of test, but can reconsider if reviewers disagree.